spec/components/citation_component_spec.rb

```diff
@@ -56,5 +57,46 @@
         end
       end
     end
+
+    context 'when given an instrument pathway' do
+      let(:work_version) {
+        build(
+          :work_version,
+          :published,
+          work: build(:work, work_type: 'instrument', doi: '10.26207/123'),
+          title: 'Citation Title',
+          published_date: '2024-02-16',
+          doi: '10.26207/123'
+        )
+      }
+      let(:authorship1) { build :authorship, given_name: 'Alan', surname: 'Grant' }
+      let(:citation_component) { render_inline(component) }
+      let(:data_and_code) { false }
+      let(:instrument) { true }
+
+      before { work_version.creators = [authorship1] }
+
+      context 'when there is one creator' do
+        let(:expected_citation) { 'Grant, Alan (2024). Citation Title [Data set]. Scholarsphere. https://doi.org/10.26207/123' }
+
+        it 'renders the citation' do
+          expect(citation_component.css('div.keyline')).not_to be_empty
+          expect(citation_component.text).to include(expected_citation)
+        end
+      end
+
+      context 'when there are multiple creators' do
+        let(:expected_citation) { 'Grant, Alan; Sattler, Ellie; Malcolm, Ian (2024). Citation Title [Data set]. Scholarsphere. https://doi.org/10.26207/123' }
```

Comment:
There's uh, another example. See, here, now I'm sitting by myself, uh, uh, talking to myself. That's-that's chaos theory.